config/initializers/job_execution.rb

```diff
@@ -1,3 +1,5 @@
+require 'rbtrace'
```

Comment:
Let's wrap this in an environment variable check, so people running Samson can easily enable/disable this functionality without needing to modify the code.